workspaces/spectacle/src/graphql/schema.ts

```diff
@@ -258,10 +258,7 @@ type ChangesResult {
 """
 Endpoint Changes
 """
-type EndpointChanges {
-  # URL for Optic change documentation
-  opticUrl: String
```

Comment:
This isn't called anywhere, and isn't returned in the buildEndpointChanges function